src/components/pages/ChildDashboard/RenderChildDashboard.js

```diff
@@ -66,7 +66,11 @@ const RenderChildDashboard = props => {
             sm={12}
             onClick={handleAcceptMission}
           >
-            <p className="accept-mission-text">ACCEPT THE MISSION!</p>
+            {submissionData.length > 0 ? (
```

Comment:
Does this only check the length of the SubmissionData array?  Will it need to check the timestamp of the submission in the future?